packages/salesforcedx-utils-vscode/test/jest/services/sourceTrackingService.test.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { SourceTracking } from '@salesforce/source-tracking';
+import { SourceTrackingService } from '../../../src/services';
+
+jest.mock('@salesforce/core', () => ({
+  ...jest.requireActual('@salesforce/core'),
```

Comment:
oh this is interesting! What benefit did `requireActual` provide here?

Reply:
Working through these unit tests was an exercise in mocking lots of dependencies.  This one mocks SDR stuff that SDR depends on.  Without it the test fails with:

<img width="500" alt="2023-03-21_15-42-21" src="https://user-images.githubusercontent.com/46458081/226758374-d7de086c-377f-4c0e-8f57-f45306ba77f3.png">